tests/resources/example_db/subjects/example_subject/sessions/example_session/example_session.json

```diff
@@ -28,5 +28,28 @@
   "attrs": {},
   "date_modified": "2024-04-09 11:27:30",
   "virtual_fit_approval_for_target_id": "example_target",
-  "transducer_tracking_approved": false
+  "transducer_tracking_results": [
+    {
+      "photoscan_id": "example_photoscan",
+      "transducer_to_photoscan_transform": {
+        "matrix": [
+          [1.0, 0.0, 0.0, 0.0],
+          [0.0, 1.0, 0.0, 0.0],
+          [0.0, 0.0, 1.0, 0.0],
+          [0.0, 0.0, 0.0, 1.0]
+        ],
+        "units": "mm"
+      },
+      "photoscan_to_volume_transform": {
+        "matrix": [
+          [1.0, 0.0, 0.0, 0.0],
+          [0.0, 1.0, 0.0, 0.0],
+          [0.0, 0.0, 1.0, 0.0],
+          [0.0, 0.0, 0.0, 1.0]
```

Comment:
Could make these matrices slightly different from the  identity and different from each other, just to help make the tests slightly pickier